recipes-bsp/tegra-binaries/tegra-redundant-boot_32.3.1.bb

```diff
@@ -50,12 +52,13 @@ PACKAGES = "${PN}-nvbootctrl ${PN} ${PN}-dev"
 FILES_${PN}-nvbootctrl = "${sbindir}/nvbootctrl"
 FILES_${PN} += "/opt/ota_package"
 FILES_${PN}-dev = "${datadir}/nv_tegra/rollback"
-SYSTEMD_SERVICE_${PN} = "nv_update_verifier.service"
+SYSTEMD_SERVICE_${PN} = "${@bb.utils.contains('DISTRO_FEATURES', 'systemd', 'nv_update_verifier.service', '', d)}"
 SYSTEMD_SERVICE_${PN}_tegra210 = ""
 RDEPENDS_${PN}-nvbootctrl = "setup-nv-boot-control"
 RDEPENDS_${PN}-nvbootctrl_tegra210 = ""
 ALLOW_EMPTY_${PN}-nvbootctrl_tegra210 = "1"
-RDEPENDS_${PN} = "${PN}-nvbootctrl setup-nv-boot-control-service"
-RDEPENDS_${PN}_tegra210 = "setup-nv-boot-control-service python-argparse python-core python-subprocess python-textutils python-shell"
+RDEPENDS_${PN} = "${PN}-nvbootctrl ${@bb.utils.contains('DISTRO_FEATURES', 'systemd', 'setup-nv-boot-control-service', '', d)}"
+RDEPENDS_${PN}_tegra210 = "${@bb.utils.contains('DISTRO_FEATURES', 'systemd', 'setup-nv-boot-control-service', '', d)} \
```

Comment:
See above.  The right way to fix this is to provide an initscript for the setup-nv-boot-control service, so it will work on both systemd and sysvinit builds.

Reply:
This change is needed, otherwise it will create a dependency on "mender-tegra-bup-payload-install" that is not available on a sysvinit system.